### ISSUE TITLE
Missing sound effects

### ISSUE THREAD

**Issue by Warr1024** [Irevol/minetest-citadel#34]
Sound effects that probably _should_ be in the game but are not:

- ~~Footsteps - Fixed in #36~~
- Picking up items (plants, stones, treasures)
- Opening the letter/book (thanks Wuzzy, below)
- ~~Unbarring sound effect - Fixed in #36~~
- Maybe a special sound for time change *failure*

Not necessarily obvious but worth considering:

- Make nearby collectables emit sounds to thematically match particle effects (thrumming power) and make them a bit easier to spot.

Nice to have:

- "Tap" sounds (punching a node, even if you can't dig it; see piranesi_redo or klots games)

**Comment by Wuzzy2** [comment]
You and your tap sound. XD
You really want to put that into every game, don't you? XD

But the other suggestions are all reasonable.

**Comment by Wuzzy2** [comment]
Additional sound suggestion: Opening the letter

**Comment by Warr1024** [comment]
> You and your tap sound. XD
> You really want to put that into every game, don't you? XD

Yes.  I like consistency.  Diggable nodes make sounds when you start to dig them, so it's something you learn to expect from other games, including many in MT.  I also found it's really pivotal for blindfolded speedrunning, which I'd eventually like to do with Citadel.  They should really add a "punch" sound for nodes (especially if it can be played client-side) for when nodes are not diggable and thus can't use the "dig" sound.
